7.0.x-qa-30323

Hanlf opened the pull request:
https://issues.liferay.com/browse/LRQA-30323
ee-7.0.x only
CC @GinsonRen

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/191/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 1 hour 52 minutes 22 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0d9288a4d909219b93c55eb4a2f290efec6f88ae">0d9288a4d909219b93c55eb4a2f290efec6f88ae</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/191/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13919/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15017/">test-portal-acceptance-pullrequest-batch(master)/functional-jboss64-mysql56-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15019/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13506/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14854/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/1</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14853/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12385/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15421/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14576/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/10</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15676/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/11</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15811/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/12</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15168/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/13</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15167/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/14</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14200/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/15</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14199/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/16</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15169/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/17</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15813/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/18</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15814/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/19</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14991/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14201/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/20</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15170/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/21</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15171/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/22</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14202/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/23</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14203/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/24</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15002/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/25</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15173/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/26</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15003/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/27</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15817/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/28</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15818/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/29</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15675/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/3</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15006/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/30</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15004/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/31</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15174/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/32</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14011/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/33</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14204/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/34</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15023/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/35</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15677/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/36</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15008/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/37</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15009/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/38</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14857/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/39</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13510/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/4</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15678/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/40</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14856/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/41</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15819/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/42</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15176/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/43</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12387/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/44</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14855/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/5</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14489/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/6</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14990/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15420/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12386/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/9</a></li><li><strike><strong><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14012/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/0</a></strong></strike></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13370/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13922/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14492/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15054/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14423/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14493/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15423/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14424/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15055/">test-portal-acceptance-pullrequest-batch(master)/functional-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14426/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13371/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/217/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-13.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/191/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/191/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>63 Jobs Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="FAILURE"><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14012/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/0</a></h5><h6>Job Results:</h6><p>0 Tests Passed.<br />1 Test Failed.</p><ol><li><strong><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/14012/">test-portal-acceptance-pullrequest-batch&#40;master&#41; ?? 0,!master #14012</a></strong><pre><code>     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:237: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:582: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-tomcat\.xml:48: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-tomcat\.xml:56: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:8236: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:2822: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:2918: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:3336: No server is available at http://localhost:8080/web/guest
      \[get\] Getting: http://test\-1\-9/job/test\-portal\-acceptance\-pullrequest\-batch\(master\)/AXIS\_VARIABLE=0,label\_exp=\!master/14012//consoleText
      \[get\] To: /opt/dev/projects/github/liferay\-portal/20160624110742107\.txt
   \[delete\] Deleting: /opt/dev/projects/github/liferay\-portal/20160624110742107\.txt
     \[exec\] 24\-Jun\-2016 18:07:42\.313 INFO \[main\] org\.apache\.catalina\.core\.StandardServer\.await A valid shutdown command was received via the shutdown port\. Stopping the Server instance\.
     \[exec\] 24\-Jun\-2016 18:07:42\.314 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.pause Pausing ProtocolHandler \["http\-nio\-8080"\]
     \[exec\] 24\-Jun\-2016 18:07:42\.365 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.pause Pausing ProtocolHandler \["ajp\-nio\-8009"\]
     \[exec\] 24\-Jun\-2016 18:07:42\.415 INFO \[main\] org\.apache\.catalina\.core\.StandardService\.stopInternal Stopping service Catalina
     \[exec\] 24\-Jun\-2016 18:07:42\.429 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.stop Stopping ProtocolHandler \["http\-nio\-8080"\]
     \[exec\] 24\-Jun\-2016 18:07:42\.432 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.stop Stopping ProtocolHandler \["ajp\-nio\-8009"\]
     \[exec\] 24\-Jun\-2016 18:07:42\.433 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.destroy Destroying ProtocolHandler \["http\-nio\-8080"\]
     \[exec\] 24\-Jun\-2016 18:07:42\.433 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.destroy Destroying ProtocolHandler \["ajp\-nio\-8009"\]</code></pre></li></ol></li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

brianwulbern commented:
ci:retest

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/194/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 50 minutes 11 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ba360d30d3c8fea9bfcdc12be4daf5bbd00a66ef">ba360d30d3c8fea9bfcdc12be4daf5bbd00a66ef</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/194/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13704/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13705/">test-portal-acceptance-pullrequest-batch(master)/functional-jboss64-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15499/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13707/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13706/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14495/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13589/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15500/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14653/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/10</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15104/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/11</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15751/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/12</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14497/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/13</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13590/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/14</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15503/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/15</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15502/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/16</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15070/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/17</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12456/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/18</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15752/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/19</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15068/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15885/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/20</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14011/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/21</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15111/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/22</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14498/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/23</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13710/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/24</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13592/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/25</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13593/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/26</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15118/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/27</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12457/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/28</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14012/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/29</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13709/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/3</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15753/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/30</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15450/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/31</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15112/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/32</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14581/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/33</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15261/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/34</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13441/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/35</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12458/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/36</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14014/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/37</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15754/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/38</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15120/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/39</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15501/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/4</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15119/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/40</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15451/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/41</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15113/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/42</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14013/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/43</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13442/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/44</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13591/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/5</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14496/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/6</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15069/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13708/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15884/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/9</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15121/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15114/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14015/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/2</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15756/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12460/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/1</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15122/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12461/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15886/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15504/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/2</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15757/">test-portal-acceptance-pullrequest-batch(master)/functional-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12462/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15071/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/217/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-13.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/194/jenkins-report.html">here</a>.</h5><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

brianwulbern commented:
No test changes needed :)

brianwulbern commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/41021
https://github.com/brianchandotcom/liferay-portal/pull/41022

Commit:
LPS-66816 Edit the staging group site settings when staging is active

juliocamarero committed:
LPS-67751 Adds missing configuration bean declaration

juliocamarero committed:
LPS-67751 Move missing configuration from portlet properties to config admin and fix typo

Conflicts:
	modules/apps/web-experience/journal/journal-service/src/main/java/com/liferay/journal/configuration/JournalServiceConfiguration.java

Ithildir committed:
LPS-66396 Test Maven

Commit:
LPS-67892 Language property added for document comment activity

Commit:
LPS-67892 Add final period.

Commit:
LPS-67892 Don't include add comment in Social Activity because we don't have anything available for DM and it's not necessary for this fix

brianchandotcom committed:
LPS-67892 Sort

brianchandotcom committed:
LPS-67892 lang keys

rafaprax committed:
LPS-69139 Create Blade Template for Form Field Type

rafaprax committed:
LPS-69139 Adding test

samanzanedo committed:
LPS-67751 Add language key. Also translate to Spanish.

samanzanedo committed:
LPS-67751 buildLang

samanzanedo committed:
LPS-67751 Baseline

ericyanLr committed:
LPS-70013 prevent menu items to be added to the Add menu when configuring a page template, for all portlets using the BasePortletToolbarContributor, not just the asset publisher portlet

ericyanLr committed:
LPS-70013 prevent menu items to be added to the Add menu on a staging live site when the staging.live.group.locking.enabled property is set to false, for all portlets using the BasePortletToolbarContributor, not just the asset publisher portlet

ericyanLr committed:
LPS-70013 extend BasePortletToolbarContributor instead of directly implementing PortletToolbarContributor

ericyanLr committed:
LPS-70013 prevent menu items to be added to the add menu for the Asset Publisher portlet when viewing a page that inherits changes from a page template

Commit:
LPS-69815 NPE when upload a file through WebDAV under workflow with sync deployed

ericyanLr committed:
LPS-70013 prevent menu items to be added to the add menu for the Web Content Display portlet when viewing a page that inherits changes from a page template

IstvanD committed:
LPS-67892 Activity is now shown for document comments

gamerson committed:
LPS-69139 add author in to address new SF requirements

gamerson committed:
LPS-69139 update dependencies

gamerson committed:
LPS-69139 just add gradle tests now, maven will come later

Hanlf committed:
LRQA-30159 Update the value of vars in description message

Hanlf committed:
LRQA-30159 Add conditional to choose a persona

IstvanD committed:
LPS-67892 TYPE_ADD_COMMENT for DLFileEntry should be among the social activity definitions, otherwise it will not be created (reverting 2b13792b452b4d61fc83d0b8fad530e02723b97c)

Commit:
LPS-69815 Deleting dlSyncEvent in another thread while DLApp is updating dlSyncEvent can cause org.hibernate.StaleStateException. Exposed when using WebDAV + workflow. Instead, delete during scheduled maintenance task.

Commit:
LPS-69815 WebDAV will create a PWC before creating an approved version

Ithildir committed:
LPS-66396 Create JUnit rule to run Maven tests

daledotshan committed:
LPS-70161 use timezone id

ealonso committed:
LPS-70134 Removes backURL

ealonso committed:
LPS-70134 Redirects to view layout url

Hanlf committed:
LRQA-30226 Clicking the tab instead of using gotoPage macro

Ithildir committed:
LPS-66396 cache

Ithildir committed:
LPS-66396 Add missing annotation

Ithildir committed:
LPS-66396 Use Maven Executor

dustinryerson committed:
LPS-67751 Baseline

dustinryerson committed:
LPS-67751 SF

jbalsas committed:
LPS-70202 Update metal packages to the latest released versions

gamerson committed:
LPS-69139 update to latest gradle-plugins

codyhoag committed:
LPS-65067 Readd original LPKG Javadoc

Ithildir committed:
LPS-69963 SF

Ithildir committed:
LPS-69963 Allow to pass an empty cache dir

nikki-pru committed:
LPS-69484 Add paths for viewing labels and values in KB Articles

Ithildir committed:
LPS-69963 Make bundles cache dir configurable

Ithildir committed:
LPS-69963 Always test with a fresh bundles cache dir

nikki-pru committed:
LPS-69484 Add macros for adding and editing custom fields in KB articles and folders

nikki-pru committed:
LPS-69484 Add new tests for adding and editing KB articles and folders with custom fields

Ithildir committed:
LPS-69139 SF

shuyangzhou committed:
LPS-70280 User id can never be negative

shuyangzhou committed:
LPS-70280 Faster string operations

codyhoag committed:
LPS-65067 Edit Javadoc

codyhoag committed:
LPS-65067 format Javadoc

codyhoag committed:
LPS-65067 Make clear that map value is LPKG's app bundle list

jbalsas committed:
LPS-70202 Update npm-shrinkwrap (auto-generated)

mtambara committed:
LPS-65067 Verify will only throw LPKGVerifyException if a bundle exists with same symbolic name and version

Commit:
LPS-69865 Link to page field's is cached with the same value in a web content in different virtual hosts

sherlyliu committed:
LPS-69402 Update js selector

Ithildir committed:
LPS-69501 Use the published portal tools by default

Ithildir committed:
LPS-69501 Update changelog

Ithildir committed:
LPS-69501 This is already the default value now

Ithildir committed:
LPS-69501 Use the locally deployed Service Builder

shuyangzhou committed:
LPS-70074 Remove _HAS_SESSION_DELIMITER field

Preston-Crary committed:
LPS-70231 Inline product_menu_control_menu_entry_icon.jsp.

Ithildir committed:
LPS-69962 Log download progress

Ithildir committed:
LPS-69962 Fix Ant sample

Ithildir committed:
LPS-69962 Add packageinfo

shuyangzhou committed:
LPS-70180 Use setLocalizedContextAttribute()

ctampoya committed:
LRQA-29690 Add EE only Kaleo tests to PR tester

Commit:
LPS-69266 Update the doc accordinly. We will update it once we finally move to Hikari

xbrianlee committed:
LRQA-30159 SF

brianchandotcom committed:
LPS-69501 prep next, use latest

brianchandotcom committed:
LPS-69501 cache

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-js/.gitrepo'.

Ithildir committed:
LPS-69501 Fix changelog

david-truong committed:
LPS-70233 test building theme against a war file

david-truong committed:
LPS-70233 support wars as parentFile and filter out compiled css files

david-truong committed:
LPS-70233 consolidate copy theme logic

ambrinchaudhary committed:
LPS-70262 Update AlloyEditor to version 1.3.1

Ithildir committed:
LPS-70233 SF

sharonchoi committed:
LRQA-30241 Add 'osgi.app.includes' property for Social Bookmarks tests

Ithildir committed:
LPS-70233 Be sure that parent themes are correctly passed

shuyangzhou committed:
LPS-70231 SF

Ithildir committed:
LPS-70233 cache

Hanlf committed:
LRQA-30323 Remove the commits form LRQA-29987

Hanlf committed:
LRQA-30323 Remove unnecessary step to assert product menu is closed

brianchandotcom committed:
LPS-69815 SF

brianchandotcom committed:
artifact:ignore osgi-felix-util 1.0.0 prep next

brianchandotcom committed:
artifact:ignore osgi-felix-util 1.0.0 artifact properties

brianchandotcom committed:
artifact:ignore osgi-felix-util 1.0.0 change log

brianchandotcom committed:
osgi-felix-util 1.0.0 packageinfo

brianchandotcom committed:
artifact:ignore osgi-felix-util 1.0.0 prep next

brianchandotcom committed:
artifact:ignore osgi-felix-util 1.0.0 artifact properties

brianchandotcom committed:
osgi-felix-util 1.0.0 apply

brianchandotcom committed:
build.gradle auto SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-css/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-editor/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-js/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-configuration/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-language/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-remote/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/sync/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/journal/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/layout/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/product-navigation/.gitrepo'.

Ithildir committed:
LPS-69494 cache

brianwulbern commented:
Will use https://github.com/brianwulbern/liferay-portal/pull/178

Closing this one.
